frontend/fingertips-frontend/components/molecules/Inequalities/LineChart/Table/index.tsx

```diff
@@ -77,16 +53,10 @@ const getCellHeader = (heading: string, index: number): ReactNode => {
 };
 
 export function InequalitiesTable({
```

Comment:
This function name (and the name of the props type) should probably be updated to reference the fact it's a line chart table now too:
```suggestion
export function InequalitiesLineChartTable({
```